### PR TITLE
Create devex-survey.md

### PR DESCRIPTION
**Pull request by reese-lee**
This PR is to create a banner announcement about the just-launched [Developer Experience survey](https://docs.google.com/forms/d/1orPz5ayzosFrgYRm3-y90UMrt2ZjvIBKMDL_a2E3Fq8/edit?usp=sharing_eil_se_dm&ts=67575cca), which is open through Jan 31, 2025. 

Please lmk if I did this correctly or not 😅

### CODE REVIEW

**Comment by reese-lee** [comment]
@open-telemetry/sig-end-user-maintainers

File: content/en/announcements/devex-survey.md
Comment (tiffany76) [inline content/en/announcements/devex-survey.md:9]:
```suggestion
<i class="fas fa-bullhorn"></i> It takes just a few minutes to 
help us [improve the developer experience](https://docs.google.com/forms/d/1orPz5ayzosFrgYRm3-y90UMrt2ZjvIBKMDL_a2E3Fq8/viewform?ts=67575cca&edit_requested=true)!
Survey is open through January 31, 2025.
```

**Review by tiffany76 — COMMENTED** [review]
The file set up LGTM. I suggested a couple wording changes. Thanks!

File: content/en/announcements/devex-survey.md
Comment (chalin) [inline content/en/announcements/devex-survey.md:9]: Here's a suggestion to shorten the banner. I think it's useful to mention the "5-10 min" estimate:

```suggestion
<i class="fas fa-bullhorn"></i> Help us [improve the developer experience] (5-10
min). Survey is closes January 31.

[improve the developer experience]:
  https://docs.google.com/forms/d/1orPz5ayzosFrgYRm3-y90UMrt2ZjvIBKMDL_a2E3Fq8/viewform?ts=67575cca&edit_requested=true
```

**Review by svrnm — APPROVED** [review]

**Review by chalin — APPROVED** [review]
LGTM once all checks pass.

File: content/en/announcements/devex-survey.md
Comment (tiffany76) [inline content/en/announcements/devex-survey.md:8]:
```suggestion
min). Survey closes January 31.
```

**Review by tiffany76 — APPROVED** [review]
One tiny fix!

**Pull request merged**
